datafusion/src/physical_plan/mod.rs

```diff
@@ -89,6 +89,34 @@ impl Stream for EmptyRecordBatchStream {
 /// Physical planner interface
 pub use self::planner::PhysicalPlanner;
 
+/// Statistics for an physical plan node
```

Comment:
At some point it probably makes sense to put these into their own module (aka `pub mod stats` or something). Doesn't have to be for this PR

Reply:
Agreed! I plan to make a separate PR to also allow physical expressions to propagate statistics (see #992). That will probably be the right moment to refactor this.